src/js/utils/trycatch.ts

```diff
@@ -14,7 +20,7 @@ export function tryCatch(fn, ctx, args = []) {
     }
 }
 
-export function JwError(name, error) {
+export function JwError(this: JWErrorInt, name: string, error: Error): void {
```

Comment:
Can you describe or link to docs about the `this` argument added above?

< before 
```js
function JwError(name, error)
```
^after
```js
function JwError(this, name, error)
```

Reply:
https://www.typescriptlang.org/docs/handbook/functions.html#this-parameters